dev.neovide.neovide.yml

```diff
@@ -0,0 +1,50 @@
+id: dev.neovide.neovide
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# sdk-extensions:
+#   - org.freedesktop.Sdk.Extension.rust-stable
+command: neovide
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri # opengl
+  - --share=network
+  - --share=ipc
+  - --filesystem=host
+  - --filesystem=xdg-config/neovide
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
instead you should build vim here and run it in the sanbox. Basially there is no point sanboxing this otherwise.

Reply:
That would be possible, but there are two big drawbacks to that approach which motivated this one.
1. a sandboxed nvim instance can't access host binaries, which break the functionality of numerous plugins (git, lsp servers, linters, etc)
2. no access to pre-existing user configs for neovim.

Reply:
This isn't allowed by default, you have to ask for an exception https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access